drivers/volume/linstor/linstor.go

```diff
@@ -268,9 +271,13 @@ func (l *linstor) GetPodVolumes(podSpec *v1.PodSpec, namespace string, includePe
 				continue
 			}
 
-			if pvc.Status.Phase == v1.ClaimPending {
-				return nil, nil, &storkvolume.ErrPVCPending{
-					Name: volume.PersistentVolumeClaim.ClaimName,
+			if !includePendingWFFC && pvc.Status.Phase == v1.ClaimPending {
+				if includePendingWFFC && isWaitingForFirstConsumer(pvc) {
```

Comment:
This will never evaluate to true because we only enter if includePendingWFFC is false from the above if statement. Review logic

Reply:
Yeah, sorry. Just a little copy/paste bug here 😅